servant-auth/servant-auth/servant-auth.cabal

```diff
@@ -35,7 +35,7 @@ library
   build-depends:
       base                    >= 4.10     && < 4.19
     , containers              >= 0.6      && < 0.7
-    , aeson                   >= 1.3.1.1  && < 3
+    , aeson                   >= 2.0      && < 3
```

Comment:
@ysangkok is this really necessary?

Reply:
the version of `jose` we use doesn't permit aeson 1.x. So I think it is better to set the tighten the bound here to reflect what can actually be chosen. So no, it is not necessary, but having 1.3.1.1 is confusing, and therefore I think we should get rid of it.